README.md

```diff
@@ -157,3 +157,42 @@ To use the provided configuration file in your Eclipse, just add this line the p
 ```
 -Drudder.configFile=<path-to-rudder-tests>/dev/configuration.properties
 ```
+
+Using aws to provide hosts
+------------------------
+
+It is possible to use rtf with aws. It requires the plugin aws.
+```
+vagrant plugin install vagrant-aws
+```
+
+To deploy aws instances with rtf you need define aws as your vagrant provider.
+This can be done easily:
+```
+export VAGRANT_ACTIVE_PROVIDER=aws
+```
+
+Vagrant does not support multi-provider setup in the same Vagrantfile, make sure to comment all your non-aws instances in your Vagrantfile before executing any rtf command.
+
+To ensure the communication between aws instances you must:
+  -create a VPC for your rudder instances
+  -create 2 security groups inside your VPC with the following
+      * server: Inbound ports 22(ssh mandatory for vagrant install)/80/443/514(reporting)/5309
+                Default reporting mode is in UDP, not TCP.
+                If you are thinking of tunning ubuntu servers with an 4.1 or older agent you will most likely need to also open the port 5514
+      * agent: Inbound ports 22/5309
+
+Create them in the VPC panel => security_group, not in the classic EC2 board.
+
+Configure the parameters at the top of the vagrantfile.rb:
+```
+$aws_key='xxxxxxxxxxxxxxxxxxxx'
+$aws_secret='xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
+# name of your key (region specific)
+$aws_keyname='my_keyname'
+$aws_vpc='subnet-c412ccbf'
+
+$aws_server_group='sg-8dcb97e4'
```

Comment:
Please describe these groups

Reply:
They should be overwritten by the user. They are described from line 179 to line 184.